sdk/cosmos/azure_data_cosmos/src/options/mod.rs

```diff
@@ -9,6 +9,19 @@ mod query_options;
 mod read_container_options;
 mod read_database_options;
 
+cfg_if::cfg_if! {
```

Comment:
This is the usage of `cfg_if`. Without it, we'd have to repeat the `#[cfg]` attribute on each individual `mod` and `pub use` line.